cabal-install/src/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -138,7 +138,7 @@ import qualified Distribution.Client.BuildReports.Anonymous as BuildReports
 import qualified Distribution.Client.BuildReports.Storage as BuildReports
          ( storeLocal )
 
-import           Distribution.Client.Config (getCabalDir)
```

Comment:
I’m silly: how does it work that the import list is empty and yet you’re able to use e.g. mkCabalDirLayout? Empty import is suspicious anyway.